critiquebrainz/frontend/messages.pot

```diff
@@ -1,14 +1,14 @@
 # Translations template for PROJECT.
-# Copyright (C) 2021 ORGANIZATION
+# Copyright (C) 2022 ORGANIZATION
```

Comment:
I don't think there are translation updates to apply as only metadata appears to change. So I suggest to revert changes to the file.

Reply:
There was a new column `ratings` added, so I added it for translation. Also, due to the addition of code, line numbers had changed.  So, I updated the file.

Reply:
Ah! This change is fine then.